rom/scripts/GREEDY.md

```diff
@@ -0,0 +1,33 @@
+To use the greedy algorithm obtained from algorithm 2 of Choi et. al's
+paper "Gradient-based constrained optimization using a database
+of linear reduced-order models": https://arxiv.org/abs/1506.07849
+to build a ROM database with Laghos, use one of the following commands.
+
+Hyperreduction is not supported!
```

Comment:
Why not? Would the online simulations be faster with hyperreduction, or does it not matter since FOM simulations are needed anyway?

Reply:
This is coming in the next PR.